packages/astro/test/units/routing/manifest.test.js

```diff
@@ -26,6 +26,18 @@ function getLogger() {
 	};
 }
 
+function checkRouteRelations(routes, relations) {
```

Comment:
Since this function runs an assertion, it's custom to use a name that hints that. So I would call it `assertRouteRelations`. We are about to move to `node:test`, which uses `assert`.